frontend-project/src/pages/cases/CasesDetailView.tsx

```diff
@@ -46,11 +45,11 @@ function CasesDetailView({
   const [isSubmitting, setIsSubmitting] = useState(false);
   const editedCase = cases.data.find(value => value.id === Number(match.params.id));
   const [form] = Form.useForm();
-  (window as any).se = smallEodSDK;
```

Comment:
W jednym PR to wprowadzamy, w innym to usuwamy. Powinniśmy ustalić czy chcemy ten kod, czy nie.

Reply:
Nie chcemy. Pozostały kod do debugowania.